resources/js/processes/screens/index.js

```diff
@@ -29,6 +39,43 @@ new Vue({
                     direction: 'desc'
                 }
             ]);
+        },
+        resetFormData() {
+            this.addScreen.formData = Object.assign({}, {
```

Comment:
Since the assignment is to an empty object, wouldn't it be cleaner to do the assignment directly? I mean:
this.addScreen.errors = {title:null, type: null, descrtiption: null}